deploy-workflow/workflows/sdk/.github/workflows/main.yml

```diff
@@ -0,0 +1,102 @@
+# v1.0.0
+name: CI
+
+on:
+  workflow_dispatch:
+  push:
+    paths-ignore:
+      - '.github/**'
+      - 'docs/**'
+      - 'README.md'
+      - 'LICENSE'
+    branches: [ main ]
+
+jobs:
+  CI:
+    runs-on: ubuntu-latest
+    env:
+      GITHUB_TOKEN: ${{ secrets.REPO_TOKEN }}
+      BLOB_SAS: ${{ secrets.BLOB_TOKEN }}
+      VERSION_SUFFIX: ""
+      ARTIFACT_URL: ""
+      PACKAGE_SERVER: "ghcr.io/virtocommerce/"
+      DEMO_MANAGER_CONTAINER: "platform-demo-manager"
+      UI_KIT_CONTAINER: "platform-ui-kit"
+
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+
+    - name: Use Node.js 14.x
+      uses: actions/setup-node@v1
+      with:
+        node-version: 14.x
+
+    - name: Install dependencies
+      run: |
+        npm install
+    - name: Get Image Version
+      uses: VirtoCommerce/vc-github-actions/get-image-version@master
+      id: image
+      with:
+        projectType: theme
+
+    - name: Set release variables
+      if: ${{ github.ref == 'refs/heads/main' }}
+      run: |
+        echo "VERSION_SUFFIX=${{ steps.image.outputs.prefix }}" >> $GITHUB_ENV
+    - name: Set release-alpha variables
+      if: ${{ github.event_name == 'workflow_dispatch' }}
+      run: |
+        echo "VERSION_SUFFIX=${{ steps.image.outputs.fullVersion }}" >> $GITHUB_ENV
+    - name: Set PR variables
+      if: ${{ github.event_name == 'pull_request' }}
+      run: |
+        echo "VERSION_SUFFIX=${{ steps.image.outputs.taggedVersion }}" >> $GITHUB_ENV
+    - name: Set ARTIFACT_URL_DM
+      run: |
+        echo "ARTIFACT_URL_DM=${{ env.PACKAGE_SERVER }}${{ env.DEMO_MANAGER_CONTAINER }}:${{ env.VERSION_SUFFIX }}" >> $GITHUB_ENV
+    - name: Set ARTIFACT_URL_UK
+      run: |
+        echo "ARTIFACT_URL_UK=${{ env.PACKAGE_SERVER }}${{ env.UI_KIT_CONTAINER }}:${{ env.VERSION_SUFFIX }}" >> $GITHUB_ENV
+    - name: Build platform-manager-demo
+      run: |
+        npm run build
+    - name: Build platform-manager-ui docs
+      run: |
+        npm run storybook-build:ui
+    - name: Docker Login
+      uses: azure/docker-login@v1
+      with:
+        login-server: ghcr.io
```

Comment:
Use environment variable for **ghcr.io**

Reply:
Done.